migrations/versions/565ea5bc3937_.py

```diff
@@ -0,0 +1,27 @@
+"""empty message
+
+Revision ID: 565ea5bc3937
+Revises: 91ee86a38001
+Create Date: 2019-02-18 15:50:18.809478
+
+"""
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '565ea5bc3937'
+down_revision = '91ee86a38001'
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('orders', sa.Column('is_billing_enabled', sa.Boolean()), server_default='False')
```

Comment:
```python
server_default=False
```

Reply:
No, this is correct

Reply:
@iamareebjamal please review

Reply:
it should have been 
```python
    op.add_column('orders', sa.Column('is_billing_enabled', sa.Boolean(), server_default='False'))
```

Reply:
@CosmicCoder96 @iamareebjamal migration fixed in #5656